doc/changelog.rst

```diff
@@ -18,6 +18,7 @@ MJX
 Python bindings
 ^^^^^^^^^^^^^^^
 - Provide prebuilt wheels for Python 3.13.
+- Removed ``nroll`` argument from rollout because its value can always be implied
```

Comment:
from rollout -> from :ref:`rollout<PyRollout>`

(apologies for the markdown, please look at the source of this comment)

Reply:
I'm asking to convert the word "rollout" to an RST link

Reply:
Like here
https://github.com/google-deepmind/mujoco/blob/e9a2f055f044fa2817b41bbc6ddd23a20f58c830/doc/changelog.rst?plain=1#L558

Reply:
Yes, got it!